flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -402,8 +399,10 @@ public void createTable() {
     public void dropTables() {
         // need to drop tables in case of retrials. Need to drop all the tables
         // that are created in test because this method is executed with every test
-        session.execute(DROP_KEYSPACE_QUERY);
-        session.execute(CREATE_KEYSPACE_QUERY);
+        synchronized (this) {
```

Comment:
I don't see how this can have any effect. Before*/After* methods are never called concurrently.

Reply:
No parallel execution of 2 `CassandraConnectorITCase` instances is possible ?

Reply:
correct.

Reply:
done